docker-compose.yml

```diff
@@ -6,14 +6,8 @@ services:
       dockerfile: Dockerfile
       target: serve
     network_mode: "host"
-    environment:
-      - SPRING83_FQDN=${SPRING83_FQDN}
-      - SPRING83_CONTACT_ADDR=${SPRING83_CONTACT_ADDR}
-    volumes: &default-logs-bind
-      - type: bind
-        source: ${SPRING83_CONTENT_DIR_HOST:?"Must define SPRING83_CONTENT_DIR_HOST as the absolute path to logs on the host."}
-        target: /content
+    env_file:
+      - .local.env
+    volumes: 
+      - ./content:/content
```

Comment:
I like the .env file thing a lot (honestly can't recall if I've ever used that!), but the removal of `SPRING83_CONTENT_DIR_HOST` won't work because `./content` isn't where I keep them on my system. I'd prefer to keep that as an env for now (just because I'm lazy and it's already in my .bashrc). Overall though I don't disagree that this isn't ideal: there has to be a better way right?